packages/containers/src/ActionBar/__snapshots__/ActionBar.test.js.snap

```diff
@@ -43,7 +43,6 @@ Object {
       "DISPLAY_MODE_SPLIT_DROPDOWN": "splitDropdown",
       "WrappedComponent": [Function],
       "compare": null,
-      "displayName": "Connect(CMF(Action))",
```

Comment:
🤔 do you know why this is removed from snapshot ?

Reply:
snapshots are not due to react, but the enzyme adapter.
I don't know why this is not displayed anymore with the new adapter